less/components/organize.less

```diff
@@ -0,0 +1,189 @@
+@import "../variables";
+
+@organize-node-bg: #DFE6ED;
+@organize-node-fg: #4B5C6B;
+@organize-branches: #788896;
+
+.organize-tree {
+    margin-bottom: 14px;
+    padding: 12px;
+
+    .top-level-list {
+        padding-inline-start: 0;
+    }
+
+    .list {
+        padding-inline-start: 26px;
+
+        & > .node-container:before {
+            content: "\a0";
+            padding: 0 4px;
+            position: absolute;
+            border-left: 2px dashed @organize-branches;
+            border-bottom: 2px dashed @organize-branches;
+            left: -14px;
+            height: 25.5px;
+        }
+
+        & > .node-container.node-middle:after {
+            content: "\a0";
+            padding: 0 4px;
+            position: absolute;
+            border-left: 2px dashed @organize-branches;
+            left: -14px;
+            bottom: 0;
+            height: ~"calc(100% - 25.5px)";
+        }
+    }
+
+    .node-container {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+
+        &.node-fake .node {
+            opacity: 0.65;
+            border: 2px dashed #788896;
```

Comment:
can we refactor the common border styles to be a re-usable mixin? e.g. maybe 
```css
.node-border {
border: 2px dashed #788896;
} 
```

and then include `.node-border;` inside your css rulesets, or alternatively (can't remember if this works)

```css
@node-border: 2px dashed #788896;
 &.node-fake .node {
border: @node-border;
}
```

Reply:
Good idea. I went with the second one (and tested that it works).